src/adhocracy_core/docs/notification.rst

```diff
@@ -43,8 +43,8 @@ Following
 On default you don`t follow any resource:
 
     >>> resp = app_participant.get(participant_path).json
-    >>> resp['data'][INotification.__identifier__]
-    {'follow_resources': []}
+    >>> resp['data'][INotification.__identifier__].get('follow_resources')
+    []
```

Comment:
As data now has "follow_resources" and "email_notification_enable" this emphasizes the point above "On default you don`t follow any resource".